src/db.ts

```diff
@@ -110,18 +110,21 @@ export interface DbOptions extends BSONSerializeOptions, WriteConcernOptions, Lo
  * @public
  *
  * @example
- * ```js
- * const { MongoClient } = require('mongodb');
- * // Connection url
- * const url = 'mongodb://localhost:27017';
- * // Database Name
- * const dbName = 'test';
- * // Connect using MongoClient
- * MongoClient.connect(url, function(err, client) {
- *   // Select the database by name
- *   const testDb = client.db(dbName);
- *   client.close();
- * });
+ * ```ts
+ * import { MongoClient } from 'mongodb';
+ *
+ * interface Pet {
+ *   name: string;
+ *   kind: 'dog' | 'cat' | 'fish';
+ * }
+ *
+ * const client = new MongoClient('mongodb://localhost:27017');
+ * const db = client.db();
+ *
+ * // Create a collection that validates our union
+ * await db.createCollection<Pet>('pets', {
+ *   validator: { $expr: { $in: ['$kind', ['dog', 'cat', 'fish']] } }
+ * })
  * ```
```

Comment:
What is the purpose of this example?

Reply:
Should option specific examples be attached to the individual options, not the class doc comment?

Reply:
I was going off of what was here before + some extra, the idea being to give an example of what this thing is for, it's the first thing you see when you land on the API page for `Db`. I just added something that gives the example some heft (an async operation that actually reaches out to mongodb)